scripts/reborn_qa_matrix/report_coverage.py

```
@@ -0,0 +1,480 @@
+    }
+
+
+def _workbook_blocked_ids(records: dict[str, dict[str, str]]) -> set[str]:
+    return {
+        test_id
+        for test_id, record in records.items()
+        if (record.get("Status") or "").lower().startswith("blocked")
+    }
+
+
+def hermetic_runner_ids(default_only: bool = False) -> set[str]:
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)